cocoapods-core.gemspec

```diff
@@ -19,22 +19,17 @@ Gem::Specification.new do |s|
   s.files         =  Dir["lib/**/*.rb"] + %w{ README.md LICENSE }
   s.require_paths =  %w{ lib }
 
-  # 6.0 requires Ruby 2.5.0
-  s.add_runtime_dependency 'activesupport', '> 5.0', '< 6'
+  s.add_runtime_dependency 'activesupport', '> 6.0', '< 7'
```

Comment:
Shouldn't this be phrased as `'~> 6'`, or am I misremembering the syntax again?
@dnkoutso what do you think?

Reply:
oops, I miss a commit when I'm doing rebase... should fix now